docs/index.md

```diff
@@ -64,5 +96,15 @@ provider "tanzu-mission-control" {
 - `client_auth_key_file` (String)
```

Comment:
Suggested adding ### Nested Schema for TMC SaaS under Tanzu Mission Control Self-Managed section. To that all details for SaaS are under its section and not be mixed.

Reply:
I get your point but we can't do that because the provider schema is defined this way. Provider is either initialised with TMC saas or TMC self managed.